e2e/framework/clients/k8s_rook_block.go

```diff
@@ -3,29 +3,30 @@ package clients
 import (
```

Comment:
There is an interface, the the tests never call a specific implementation directly, they are all wrapped in rook_test_client, which has all the operations as members, and based on the platform type, the matching implementations is used.  I am removing all 'rook' from names, but I am leaving k8s, so we can differentiate
between k8s and standalone implementations, e.g. k8s_block or standalone_block etc